### DISCUSSION TITLE
Is Javascript client 5.x compatible with SignalR .NET core 3.1?

### DISCUSSION THREAD

**Discussion by k-eb** [dotnet/aspnetcore#30031] [Q&A]
Hi, 
I have a SignalR based server application based on **.NET core 3.1** (**using Microsoft.AspNetCore.SignalR 3.1.5**).
I am now wondering whether there are any restrictions on what SignalR JavaScript client version I can use?

Concretely, is the latest version on npm ([5.0.2](https://www.npmjs.com/package/@microsoft/signalr)) compatible with SignalR on .NET core 3.1?

Thanks!

**Comment by BrennanConroy** [comment]
The client is compatible, however if new feature(s) are added, the features(s) will not work.

**Reply by k-eb** [reply]
Thanks. You mean new features in future versions of the client?

**Reply by BrennanConroy** [reply]
Yes.
